README.md

```diff
@@ -1,2 +1,40 @@
 # BitBulldozersLab
-Bluntly pushed bits, bytes, &amp; pieces for HPC
+
+**Bit Bulldozers Lab** - *Bluntly pushed bits, bytes &amp; pieces for HPC.*
+
+This repository contains proof-of-concepts (PoCs) and examples related to High Performance Computing (HPC). These PoCs and examples fulfilled some purpose when they were created, but are, in general, unmaintained and comes without any claim of fitness for a particular purpose - use at you own risk and as you see fit!
+
+## PoC / example structure
+
+The PoCs and examples are organized into a flat namespace, i.e. each folder in this repository root is a PoC / example. Each of those PoC / example folders contain all files related to that PoC / example, including a `README.md` file documenting the it.
+
+### PoC / example README.md
```

Comment:
could each readme or subtract of it be contained on the front page. Not to do now but later.